client/src/components/InputField/InputField.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import cn from 'classnames';
+
+const InputField = ({
```

Comment:
This code is from previous commits and is now merged into Master. Should be removed.

Reply:
okay will do that

Reply:
Will take a pull from the master while merging